### PR TITLE
fix: Don't prefix links to license

### LINKED ISSUE #3339: License link is not working
Steps to reproduce:
1. Load an add-on details page i.e. https://addons.allizom.org/en-US/firefox/addon/youtube-unblocker/
2. Click on the license link

Expected results:
License page is loaded.

Actual results:
A 404 old AMO desktop page is loaded.

Notes/Issues:
- on device a 404 mobile page is loaded.
Verified on FF56(Win 7, Android 7.2.1).
Issue is reproducing on both stage and dev.
Screenshot for this issue:

### PR DESCRIPTION
**Pull request by tofumatt**
Fix #3339.

### CODE REVIEW

**Comment by tofumatt** [comment]
Hmm, sure thing!

**Comment by tofumatt** [comment]
Alrighty, tests added!

File: tests/unit/amo/components/TestAddonMoreInfo.js [hunk 1]
```diff
@@ -140,12 +140,27 @@ describe(__filename, () => {
...
+    expect(link).toHaveProp('href', 'http://license.com/');
+  });
+
+  it('does not prefix a license link with the add-ons URL', () => {
+    // See: https://github.com/mozilla/addons-frontend/issues/3339
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      current_version: {
+        ...fakeAddon.current_version,
+        license: { name: 'tofulicense', url: 'www.license.com/' },
```
Comment (willdurand) [inline tests/unit/amo/components/TestAddonMoreInfo.js]: As soon as you create this license, let me know. I want to fully adopt it 😛

File: tests/unit/amo/components/TestAddonMoreInfo.js [hunk 2]
```diff
@@ -140,12 +140,27 @@ describe(__filename, () => {
...
+      },
+    });
+    const root = render({ addon });
+    const link = root.find('.AddonMoreInfo-license-link');
+
+    expect(link).toHaveProp('prependClientApp', false);
+    expect(link).toHaveProp('prependLang', false);
```
Comment (willdurand) [inline tests/unit/amo/components/TestAddonMoreInfo.js]: Can we also assert that the link will be a proper link and not something pointing to `www.license.com`?

Comment (willdurand) [reply tests/unit/amo/components/TestAddonMoreInfo.js]: Or do we have such a test in the `Link` component already?

Comment (willdurand) [reply tests/unit/amo/components/TestAddonMoreInfo.js]: Maybe asserting the `href` prop would be enough.

Comment (tofumatt) [reply tests/unit/amo/components/TestAddonMoreInfo.js]: Actually the links *aren't* always proper from the API. These will work as browsers fill in the scheme... but eventually we may want to run a migration on the server to convert them and require a scheme in the future 😄

Comment (tofumatt) [reply tests/unit/amo/components/TestAddonMoreInfo.js]: I'll file a new issue to add the schema to schema-less-links in `<Link>` though; it's not required but it's nice.

Comment (tofumatt) [reply tests/unit/amo/components/TestAddonMoreInfo.js]: Made #3345.

Comment (willdurand) [reply tests/unit/amo/components/TestAddonMoreInfo.js]: 💪

**Pull request merged**